src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -59,7 +59,7 @@ public class CustomAsyncResult : IAsyncResult
     public static class Helpers
     {
         public static bool IsWindowsImplementation { get; } = PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
-        public static bool IsNotWindowsImplementation => !IsWindowsImplementation;
+        public static bool IsNotWindowsImplementation => !IsWindowsImplementation && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
@CIPop @stephentoub I thought it was odd too.  I think what this means that that we should simplify the IsWindowsImplementation and then move IsNotOneCoreUAP into the places where the tests were relying on it.

The question is whether this is the PR to do this or not.  We want to enable the portable stuff in CI by default ASAP.